checkov/common/sca/output.py

```diff
@@ -293,33 +297,122 @@ def add_to_reports_cves_and_packages(
                     vulnerability_details={
                         "package_name": package["name"],
                         "package_version": package["version"],
-                        "licenses": format_licenses_to_string(licenses_per_package_map[get_package_alias(package["name"], package["version"])]),
+                        "licenses": format_licenses_to_string(
+                            licenses_per_package_map[package_alias]),
                         "package_type": get_package_type(package["name"], package["version"], sca_details),
                     },
                 )
             )
 
+    for root_package_index in root_packages_list:
+        deps = []
```

Comment:
```suggestion
        vulnerable_dependencies = []
```
since its not all the dependencies, only vulnerable dependencies